### DISCUSSION TITLE
Poor "focal point" for each release in UI refresh (plus a couple other things)

### DISCUSSION THREAD

**Discussion by MHLoppy** [community/community#6451] [Other Feature Feedback, Questions, & Ideas] [answered]
In the current/old releases UI, every release has a clear visual anchor which the eyes are drawn to (see image).

![image](https://user-images.githubusercontent.com/12670674/137639731-5991e095-03de-4059-bfb9-2fd2bfa1d585.png)

In the new UI, these anchoring elements are spread around a lot more, and the removal of the color from the title I think also contributes to the feeling of my eyes not "knowing" where to look at first. It sounds slightly bizarre to say, but it feels like I'm struggling to subconsciously navigate between all of text (headers vs body) because it feels too same-y as one color with no/few other identifiers other than basically just size (maybe I'd get used to it?). The tags e.g. "Latest" help but are obviously not always present.

![image](https://user-images.githubusercontent.com/12670674/137639890-6a130b07-78a5-4414-92d2-78a787cf196a.png)

---

What I did like:
- Reactions moving to the bottom and also not getting 
- Clearer separation between each individual release's "bits" - in particular the assets of each release feel much more contained within that release rather than sort of bleeding into the "not part of that release" area of the page.

**Comment by MylesBorins** [comment, marked as answer]
hey @MHLoppy, thanks for the feedback. We just shipped an update that brings the content from the left column closer and I think this fixes the disconnect you are talking about. For now our intent is to leave the title as black with blue on hover, similar to the titles we use elsewhere on github.com such as PRs and Issues
